Dockerfile

```diff
@@ -194,8 +235,8 @@ RUN TS_VERSIONS="1.6.0 1.6.1 1.7.0 1.7.1 1.7.2 1.7.3 1.7.4 1.7.5 2.0.0-rc3 2.0.0
             && cd /build/timescaledb && git reset HEAD --hard && git clean -f -d -x && git checkout ${ts} \
             && rm -rf build \
             && if [ "${ts}" = "2.2.0" ]; then sed -i 's/RelWithDebugInfo/RelWithDebInfo/g' CMakeLists.txt; fi \
-            && PATH="/usr/lib/postgresql/${pg}/bin:${PATH}" ./bootstrap -DTAP_CHECKS=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo -DREGRESS_CHECKS=OFF -DGENERATE_DOWNGRADE_SCRIPT=ON -DPROJECT_INSTALL_METHOD="${INSTALL_METHOD}"${OSS_ONLY} \
-            && cd build && make -j 6 install || exit 1; \
+            && PATH="/usr/lib/postgresql/${pg}/bin:${PATH}" ./bootstrap -DTAP_CHECKS=OFF -DWARNINGS_AS_ERRORS=off -DCMAKE_BUILD_TYPE=RelWithDebInfo -DREGRESS_CHECKS=OFF -DGENERATE_DOWNGRADE_SCRIPT=ON -DPROJECT_INSTALL_METHOD="${INSTALL_METHOD}"${OSS_ONLY} \
+            && cd build && make install || exit 1; \
```

Comment:
is `|| exit 1` needed? it just turns any non-zero exit code into 1, which is still an error. unless something relies on this being 1, I'd let it exit with the actual code it produced.

Also, instead of `cd build && make install`, you can just do `make -C build install`

Reply:
The `|| exit1` is to ensure that the `RUN` instructions ends right there; we could possibly do a `set -e` at the top though to make this shell terminate on any error.

As this is inside a for loop (over major postgresql versions), we cannot just `exit 0` here on success, we need to throw an error.